tensorflow_datasets/scripts/metadata_cleanup.py

```diff
@@ -0,0 +1,97 @@
+# Lint as: python3
+"""Removes all dataset versions from metadata directory which are not 
+   present in registered versions of TFDS.
+
+To test:
+Goto tensorflow_datasets/scripts
+
+Args : dry_run : If given, it traverse and finds all
+                 versions which are to be removed without
+                 actually removing them.
+
+             r : If given it removes all unmatched dataset versions 
+                 from metadata directory.
+```
+cd tensorflow_datasets/scripts
+
+For check not delete : python metadata_cleanup.py --dry_run
+For delete not check : python metadata_cleaup.py --r
+For both delete & check : python metadata_cleanup.py --dry_run --r
+```
+"""
+from absl import flags
+from absl import app
+import os
+import tensorflow as tf
+from tensorflow_datasets.core import registered
+
+
+FLAGS = flags.FLAGS
+flags.DEFINE_boolean('dry_run', False, "Shows all versions\
+                      which are not registered")
+flags.DEFINE_boolean('r', False, "Removes unmatched\
+                      dataset versions from metadata")
+
+__metadata_path = os.path.join(os.pardir, "testing/metadata")
+
+def _extract_metadata_versions(metadata_dir):
+    '''
+       Get all metadata direcotry versions paths,

```

Comment:
Could you format the doc string the similarly to the rest of the codebase ? And use `"""` instead of `'''`